apps/website/src/content/docs/tree.mdx

```diff
@@ -1,27 +1,160 @@
 ---
 title: Tree
-description: A tree provides a list of data.
+description: A tree provides a hierarchial lists of data with nested expandable levels.
 thumbnail: Tree
 ---
 
 <p>{frontmatter.description}</p>
+The `Tree` component can be used to organize data in an application specific way, or it can be used to
+sort, filter, group, or search data as the user deems appropriate.
```

Comment:
formatting

```suggestion

The `Tree` component can be used to organize data in an application specific way, or it can be used to sort, filter, group, or search data as the user deems appropriate.
```

Reply:
I have removed the redundant spacing.